apps/web/components/eventtype/AIEventController.tsx

```diff
@@ -157,68 +205,84 @@ const AISettings = ({ eventType }: { eventType: EventTypeSetup }) => {
           <Label>{t("your_phone_number")}</Label>
           <Controller
             name="aiPhoneCallConfig.yourPhoneNumber"
-            render={({ field: { onChange, value } }) => {
+            render={({ field: { onChange, value, name }, fieldState: { error } }) => {
               return (
-                <PhoneInput
-                  required
-                  placeholder={t("your_phone_number")}
-                  id="aiPhoneCallConfig.yourPhoneNumber"
-                  name="aiPhoneCallConfig.yourPhoneNumber"
-                  value={value}
-                  onChange={(val) => {
-                    onChange(val);
-                  }}
-                />
+                <div>
+                  <PhoneInput
+                    required
+                    placeholder={t("your_phone_number")}
+                    id="aiPhoneCallConfig.yourPhoneNumber"
+                    name="aiPhoneCallConfig.yourPhoneNumber"
+                    value={value}
+                    onChange={(val) => {
+                      onChange(val);
+                    }}
+                  />
+                  {error?.message && <ErrorMessage message={error.message} fieldName={name} />}
+                </div>
               );
             }}
           />
 
+          <>
+            <RadioGroup.Root
```

Comment:
It looks like the dot is pushing the content up and down

https://github.com/user-attachments/assets/93337914-889f-440a-9661-43e36d7fa4b5